en/releases/core_components.md

```diff
@@ -0,0 +1,56 @@
+# PX4 Core Components
+
+These core components are *guaranteed* to work with a stable point release of PX4. They form the foundation of the PX4-Autopilot and it's core functionality.
```

Comment:
What are you promising with this ["guarantee"](https://www.merriam-webster.com/dictionary/guarantee)? I mean you promise it works. Is that a guarantee that you will fix all bugs in 1 week of release? Or that there are no bugs in it?

Or to put it another way, what is the purpose of this page - is it to identify the person's responsible for the sign off of each component?

Either way, I'd use much softer words asserting the quality.

Reply:
I edited to take the comments into consideration!